package.json

```diff
@@ -93,6 +93,16 @@
                 "when": "editorTextFocus && vim.useCtrlKeys"
             },
             {
+                "key": "ctrl+a",
```

Comment:
Do the thing like we do elsewhere in package.json where we only enable these 2 things if ctrlKeys is turned on.